webcam.go

```diff
@@ -113,22 +117,29 @@ func (w *Webcam) SetImageFormat(f PixelFormat, width, height uint32) (PixelForma
 	}
 }
 
+// Set the number of frames to be buffered.
+func (w *Webcam) SetBufferCount(count uint32) error {
+    if count < 2 || count > maxBufferCount {
```

Comment:
Probably you should not limit a lower bound as well, V4l2 documentation does not mention neither lower nor upper limit on buffer count so I suggest to leave it to developer. 

Also this function does not have any effect if streaming already started, so I suggest to mention it in a doc.

Finally, according to v4l2 docs, user can actually request more buffers even if streaming already started, so as extra step would be nice to provide this feature (optionally, if you have time)

Reply:
Doing it on the fly seems tricky, so perhaps allowing it via stopping streaming, then changing the buffer count, then starting again seems a cleaner approach.